apps/docker/Dockerfile

```diff
@@ -1,38 +1,37 @@
-# Librelingo is not compatible with Node 16, using Node 14.18.3 image instead
-FROM node:14.18.3 
+FROM archlinux:base-20220227.0.49015
 
-# Setup Librelingo
+# Setup packages
+RUN pacman -Syu --noconfirm git nodejs-lts-fermium gcc yarn python3 python-pip
+
+# Setup Poetry and LibreLingo Python Packages
+RUN curl -sSL https://install.python-poetry.org | python3 - && \
+  cp /root/.local/bin/poetry /bin && \
+  pip install librelingo-yaml-loader && \
```

Comment:
we should not need this, and the next line either. we should be able to install these packages from the local repo as opposed to PyPI in the Docker image

Reply:
I will change it, to use the local ones when I have time

Reply:
amazing!